piezo_web_app/PiezoWebApp/src/handlers/submit_job.py

```diff
@@ -7,15 +7,17 @@
 class SubmitJobHandler(BaseHandler):
     @schema.validate(
         input_example={
-            'name': 'example_job_name',  # to be replaced by list of arguments we will allow the user to configure
+            'name': 'example-job-name',  # to be replaced by list of arguments we will allow the user to configure
```

Comment:
We should probably now remove this comment and add the arguments we are allowing. I believe this was originally a placeholder before we knew which arguments we would make configurable

Reply:
Given the complexity of the input arguments for this handler, it's probably best to remove the example and replace it with a link to the wiki page. See 7686a9028ac71cf809d836d31643b26e5d059f24